AnagramSolver.BusinessLogic/Classes/AnagramSolver.cs

```diff
@@ -8,49 +8,108 @@ namespace AnagramSolver.BusinessLogic.Classes
 {
     public class AnagramSolver : IAnagramSolver
     {
-        WordRepository wordRepository = new WordRepository();
+        IWordRepository wordRepository = new WordRepository();
         public List<Anagram> GetAnagrams(string command, int minLength, string filePath)
         {
             var allWords = wordRepository.GetAllWords(filePath, minLength);
 
-            // var exists = allWords.FirstOrDefault(p => p.Equals("sula"));
+            List<Anagram> newList = new List<Anagram>();
```

Comment:
var could be used here

Reply:
Also a possibility to write like that in .NET 5:
`List<Anagram> newList = new()`
However, it is a new and not popular approach yet.

Reply:
sorry about that. I think I will stick with var, more understendable